src/main/java/uk/gov/hmcts/reform/feature/config/SecurityConfiguration.java

```diff
@@ -104,8 +128,30 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers("/", "/health", "/info", "/v2/api-docs").permitAll()
                 .anyRequest().authenticated()
                 .and()
+                .formLogin().successHandler(new SuccessHandler()).permitAll()
+                .and()
+                .logout().logoutSuccessUrl("/?logout").permitAll()
+                .and()
                 .csrf().disable();
         }
+
+        private static class SuccessHandler implements AuthenticationSuccessHandler {
+
+            @Override
+            public void onAuthenticationSuccess(HttpServletRequest request,
+                                                HttpServletResponse response,
+                                                Authentication authentication) throws IOException, ServletException {
+                boolean isAdmin = authentication.getAuthorities().stream().anyMatch(authority ->
+                    authority.getAuthority().equals("ROLE_ADMIN")
```

Comment:
what about editors? (and/or other roles)
Also, perhaps a constant should be used instead of `"ROLE_ADMIN"` string?

Reply:
we only want to redirect admins to ff4j web console. otherwise - homepage